### PR TITLE
feat/modal response

### PR DESCRIPTION
**Pull request by blocdenis**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
The task goal was to make the job offers list available on mobile devices. This involves implementing a mobile version based on the provided design.

### Code Changes Review

#### src/components/Container/Container.module.scss
The changes in this file seem to be in line with the task requirements. The container has been made responsive with the addition of padding, gap and width properties for mobile view.

#### src/components/FiltersContainer/FiltersContainer.module.scss
The changes in this file are also in line with the task requirements. The width of the filter section and filter wrapper have been adjusted for mobile view. The mobile header has also been adjusted to fit the mobile screen.

#### src/components/Modal/OfferDetailsModal.module.scss
The changes in this file are extensive and seem to be in line with the task requirements. The modal has been made responsive for mobile view. The size of the company logo, title, and other elements have been adjusted for mobile view. The layout of the description and left wrapper have been changed to column-reverse for mobile view.

#### src/components/OffersContainer/OffersContainer.module.scss
The changes in this file are in line with the task requirements. The width of the offers container has been adjusted for mobile view.

#### src/components/OffersList/OffersList.module.scss
The changes in this file are in line with the task requirements. The width of the offers list has been adjusted for mobile view.

#### src/components/SearchBar/SearchBar.module.scss
The changes in this file are in line with the task requirements. The layout of the search bar has been changed to column for mobile view and its width has been adjusted.

### Conclusion
Based on the code changes provided, it appears that the task requirements have been met. The job offers list should now be available on mobile devices as per the provided design. However, without seeing the final output or having the ability to test the changes, it's hard to definitively say if everything works as expected. It's recommended to thoroughly test these changes on various mobile devices and screen sizes to ensure compatibility and responsiveness.

File: src/components/Container/Container.module.scss [hunk 1]
```diff
@@ -5,5 +5,8 @@
    @include breakpoint(tablet) {
       flex-direction: column;
       position: relative;
+      padding: 15px;
+      gap: 15px;
+      width: 100vw;
```
Comment (Sumick) [inline src/components/Container/Container.module.scss:10]: **nitpick:** I think there is not need of doing 100vw

File: src/components/FiltersContainer/FiltersContainer.module.scss [hunk 2]
```diff
@@ -15,13 +15,16 @@ section {
    width: 303px;
    // height: 594px;
    @include breakpoint(tablet) {
-      min-width: 800px;
+      // min-width: 800px;
```
Comment (Sumick) [inline src/components/FiltersContainer/FiltersContainer.module.scss:18]: **nitpick:** comment to remove

File: src/components/FiltersContainer/FiltersContainer.module.scss [hunk 3]
```diff
@@ -44,10 +47,14 @@ section {
 }
 .mobile_header {
    margin: 20px;
-   min-width: 800px;
+   // min-width: 800px;
```
Comment (Sumick) [inline src/components/FiltersContainer/FiltersContainer.module.scss:50]: nitpick: comment to remove

File: src/components/SearchBar/SearchBar.module.scss [hunk 4]
```diff
@@ -33,7 +33,8 @@
    gap: 20px;
    @include breakpoint(tablet) {
       flex-direction: column;
-      padding: 25px 10px 15px 10px;
+      // padding: 25px 10px 15px 10px;
```
Comment (Sumick) [inline src/components/SearchBar/SearchBar.module.scss:36]: nitpick: comment to remove

**Review by Sumick — APPROVED** [review]
Looks good! Few comments to remove here

**Pull request merged**